### DISCUSSION TITLE
2024-06-20(목)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#189] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by minyoung0503** [comment]
### 완료한 일
myReviewWritePage API 연결 (PR 올릴 예정)

### 해야할 일
BottomSheet 가격 계산

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 라우터 로딩 적용
- `Provider` 리렌더링 이슈 해결

### 해야할 일
- 공통 정렬 버튼 구현
- 상품 목록 API 완성되면 홈, 인기상품, 추천상품, 카테고리, 검색, 찜 작업

### 어려운 일
- 파이팅!!👍

**Comment by Seoin02** [comment]
### 완료한 일
- 닉네임 중복 검사

### 해야할 일
- 프로필 이미지 삽입 PR
- 프로필 페이지 토스트 적용
- my order 페이지 API 연동

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 엑스버튼, 실패 페이지 
- 결제 api
- toast 창 띠우기
- cardSlider 

### 해야할 일
(민영님 송은님) 완료되면 deliveryId, getBuyingId 값 불러오기
### 어려운 일

**Comment by q45402sk** [comment]
### 완료한 일
상품상세 (pr 올릴 예정)

### 해야할 일
purchaseDeliveryPage
deliveryAddPage
deliveryEdiitPage api

### 어려운 일
